tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -331,7 +331,7 @@ const performUpload = async (args: uploadResourceArgs): Promise<void> => {
 export const uploadLargeNumberOfResources = async (args: uploadResourceArgs): Promise<void> => {
   const { page, resources } = args
   await performUpload(args)
-  await page.locator(uploadInfoCloseButton).waitFor()
+  await page.locator(uploadInfoSuccessLabelSelector).waitFor()
```

Comment:
I highly doubt this was the problem. was it reproducible?

Reply:
No, couldn't reproduce locally.

Reply:
So let's not use this as a fix. Need to find what really happened during failure